staging/src/k8s.io/kubectl/pkg/cmd/describe/describe.go

```diff
@@ -62,9 +62,9 @@ var (
 		kubectl describe pods
 
 		# Describe pods by label name=myLabel
-		kubectl describe po -l name=myLabel
+		kubectl describe pods -l name=myLabel
```

Comment:
Right, but I changed it here just so that there is consistency between the examples (all other ones use "pods"). I left this change in there, but let me know if you need me to revert it